src/Maestro/tests/Maestro.ScenarioTests/EndToEndFlowLogic.cs

```diff
@@ -0,0 +1,352 @@
+using System;
+using System.Collections.Generic;
+using System.Collections.Immutable;
+using System.Linq;
+using System.Threading.Tasks;
+using Microsoft.DotNet.DarcLib;
+using Microsoft.DotNet.Maestro.Client.Models;
+using Microsoft.Extensions.Primitives;
+using NUnit.Framework;
+
+namespace Maestro.ScenarioTests
+{
+    [TestFixture]
```

Comment:
I don't see any `[Test]` attributes in here.

Reply:
I wasn't clear if I still needed it in order to ensure that it was included in the temp test folder, but taking that logic further there are a lot of other classes that would be in the same boat if that was the case. Pulled the attribute.